tests/test_nsjail.py

```diff
@@ -316,3 +318,69 @@ def test_init_args(self):
         self.assertEqual(self.nsjail.config_path, self.config_path)
         self.assertEqual(self.nsjail.max_output_size, self.max_output_size)
         self.assertEqual(self.nsjail.read_chunk_size, self.read_chunk_size)
+
+
+class NsJailCgroupTests(unittest.TestCase):
+    # This should still pass for v2, even if this test isn't relevant.
+    def test_cgroupv1(self):
+        logging.getLogger("snekbox.nsjail").setLevel(logging.ERROR)
+        logging.getLogger("snekbox.utils.swap").setLevel(logging.ERROR)
+
+        config_base = dedent(
+            """
+            mode: ONCE
+            mount {
+                src: "/"
+                dst: "/"
+                is_bind: true
+                rw: false
+            }
```

Comment:
This binding only works if the installed python version matches what you're expecting (`/usr/local/bin/python`), but will cause the test to fail if the user defines a non-default mount or exec_bin. Could we load the mounts and exec_bin from the actual config into here?

Reply:
Something like this seems to do the trick
```py
        user_config = NsJail._read_config("./config/snekbox.cfg")
        mounts = "\n".join((f"mount {{{mount}}}" for mount in user_config.mount))
        config_base = dedent(
            f"""
            {mounts}\n
            exec_bin {{{user_config.exec_bin}}}
            """
        ).strip()
```

[Patch included](https://github.com/python-discord/snekbox/files/9072325/Use_User_Config_In_Tests.txt)

Reply:
The goal of this test is to ensure NsJail does not crash when each of the cgroups is configured. If the parent cgroup is not created properly, then NsJail will crash. Which process actually runs is of no concern for this test; I just had to specify _something_ so NsJail can run. The tests run in the image, and we know the image contains Python, so using Python is a safe bet.

Reply:
This is only true for users using the provided docker image, but on local installations, or on modified versions (which is my use-case), it will error. If we don't care about the specific command being python, let's use something else that's likely to exist in all environments. Perhaps something like `ls` could be a good choice.